.github/CONTRIBUTING.MD

```diff
@@ -0,0 +1,102 @@
+# Contributing
+
+Hacking Prodigy used to be as easy as `PIXI.game.prodigy.player.gold=999`, but now we need a whole team!
+
+If you'd like to join as an official collaborator, you can message us on [our official Discord server](https://discord.gg/XQDfbfq)!
+
+Additionally, read our ([Code of Conduct](https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/blob/master/CODE_OF_CONDUCT.md)) before continuing.
+
+---
+
+## The Goal of PMGH
+
+Our main goal in this repository is to bring Prodigy's attention to their game's architecture.
+
+Most of the data stuff is handled on the client side, so hacking is as easy as changing a variable. No pwn needed!
+
+Since we've started this repository, Prodigy's security has improved by leaps and bounds, but the main issue is still the same;
+once you get to the player variable, you can easily change anything you want.
+
+Ideally everything would be handled server side. For example, arena points are already handled properly. The only way to increment arena points is by sending a "win" event, which is rate limited to once per minute.
+
+## Tasks
+
+There are lots of things to do! (Ranked from easiest to hardest)
+
+### Research
+
+As the name indicates, research. Research includes:
+
+- looking for new endpoints to mess around with
+- reading the Prodigy source extensively
+- reporting broken hacks
+- suggesting new hack ideas
+
+Requirements:
+
+- basic coding skills
+- basic understanding of what an API is (the bar is super low huh)
+- reading skills
+
+Anybody can do this! If you want to help with any of these, you can post your findings in [Discussions](https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/discussions), or [our official Discord](https://discord.gg/XQDfbfq).
+
+### Typings
+
+Basically just typescript definitions for Prodigy objects. Pretty much grunt work.
```

Comment:
an understanding of how to implement the types is also necessary

Reply:
I just copied those files from /meta, lmao

Reply:
Alr, imma do that rq

Reply:
done